### PR TITLE
[Disk Manager]: Tests: Actualize README.md

### PR DESCRIPTION
**Pull request by jkuradobery**

### CODE REVIEW

File: cloud/disk_manager/test/acceptance/test_runner/README.md [hunk 1]
```diff
@@ -1,25 +1,61 @@
...
-    ```(bash)
-    $ ya make ../../../../blockstore/tools/testing/verify-test/
-    ```
-3. Build `test-runner`:
-    ```(bash)
-    $ ya make
-    ```
-
-## "acceptance" test type
+These acceptance tests are intended to perform acceptance testing of disk-manager
```
Comment (SvartMetal) [inline cloud/disk_manager/test/acceptance/test_runner/README.md:3]: disk-manager -> Disk Manager

File: cloud/disk_manager/test/acceptance/test_runner/README.md [hunk 2]
```diff
@@ -1,25 +1,61 @@
...
-
-## "acceptance" test type
+These acceptance tests are intended to perform acceptance testing of disk-manager 
+and NBS as a part of the whole cloud. They are intended to run on a separate machine 
+with access to lab/production cluster and with internal tool `ycp` installed.
+These tests can't be launched locally.
+
+## Terminology
+
+* `Disk` - either a non-replicated disk with mirroring capabilities (basically SSD cut into chunks and shared via RDMA) or kikimr blob storage based disks. Blob storage based disks can be overlay (reference other disks).
```
Comment (SvartMetal) [inline cloud/disk_manager/test/acceptance/test_runner/README.md:10]: kikimr blob storage -> YDB Blob Storage

File: cloud/disk_manager/test/acceptance/test_runner/README.md [hunk 3]
```diff
@@ -1,25 +1,61 @@
...
+* `Snapshot` - snapshot of the disk state, stored either in S3 or in YDB with an ability to provision disks from snapshots. Snapshots can be shallow copies of other snapshots.
+* `Image` - images are the same as snapshots with the following exceptions: 
+  * Image can be created from the snapshot, snapshot can not be created from images.
+  * Disk can be created from either disk or snapshot
+  * Images can be optimized for fast provisioning, this way a pool of overlay blob-storage based disks is created. Initial image is still stored in S3 or in YDB.
```
Comment (SvartMetal) [inline cloud/disk_manager/test/acceptance/test_runner/README.md:15]: blob-storage -> YDB Blob Storage

File: cloud/disk_manager/test/acceptance/test_runner/README.md [hunk 4]
```diff
@@ -1,25 +1,61 @@
...
+with access to lab/production cluster and with internal tool `ycp` installed.
+These tests can't be launched locally.
+
+## Terminology
+
+* `Disk` - either a non-replicated disk with mirroring capabilities (basically SSD cut into chunks and shared via RDMA) or kikimr blob storage based disks. Blob storage based disks can be overlay (reference other disks).
+* `Snapshot` - snapshot of the disk state, stored either in S3 or in YDB with an ability to provision disks from snapshots. Snapshots can be shallow copies of other snapshots.
+* `Image` - images are the same as snapshots with the following exceptions: 
+  * Image can be created from the snapshot, snapshot can not be created from images.
+  * Disk can be created from either disk or snapshot
```
Comment (SvartMetal) [inline cloud/disk_manager/test/acceptance/test_runner/README.md:14]: Disk can be created from either disk or snapshot or image

File: cloud/disk_manager/test/acceptance/test_runner/README.md [hunk 5]
```diff
@@ -1,25 +1,61 @@
...
-## "acceptance" test type
+These acceptance tests are intended to perform acceptance testing of disk-manager 
+and NBS as a part of the whole cloud. They are intended to run on a separate machine 
+with access to lab/production cluster and with internal tool `ycp` installed.
+These tests can't be launched locally.
+
+## Terminology
+
+* `Disk` - either a non-replicated disk with mirroring capabilities (basically SSD cut into chunks and shared via RDMA) or kikimr blob storage based disks. Blob storage based disks can be overlay (reference other disks).
+* `Snapshot` - snapshot of the disk state, stored either in S3 or in YDB with an ability to provision disks from snapshots. Snapshots can be shallow copies of other snapshots.
```
Comment (SvartMetal) [inline cloud/disk_manager/test/acceptance/test_runner/README.md:11]: Snapshot can be incremental (based on another snapshot)

File: cloud/disk_manager/test/acceptance/test_runner/README.md (hunk 2 above)
Comment (SvartMetal) [inline cloud/disk_manager/test/acceptance/test_runner/README.md:10]: Lets not add to much details about implementation. 
Non-replicated disks can also be overlay by the way

**Review by SvartMetal — APPROVED** [review]

**Review by gy2411 — APPROVED** [review]

**Pull request merged**